### DISCUSSION TITLE
player height

### DISCUSSION THREAD

**Discussion by miembrodes** [Luke100000/minecraft-comes-alive#751] [Ideas]
I found a bug today when creating a skin, but I thought it would be a good thing if this was fixed well.

reference : https://github.com/Luke100000/minecraft-comes-alive/issues/750

You can change the height of the skin when creating the skin, but if you set it to a certain height or more, the white hitbox will reach the upper limit and stop changing. (I think about 2 blocks high is the upper limit)

I think it would be a good idea to change the stop in the middle and increase it to the upper limit.
It doesn't matter if the 2 block high space is impassable.

* Sorry for Google translation

**Comment by Luke100000** [comment]
The 2 block limitation has been added for exactly that reason, otherwise it would make gameplay near impossible.
